backend/routes/api/job-api.js

```diff
@@ -100,9 +100,42 @@ JobApi.prototype.getAllJobs = async function(req, res) {
 }
 
 JobApi.prototype.getAllJobsWithResults = async function(req, res) {
-    var userObj = req.body;
-    var objectToSend = await Helpers.getJobsWithLocations(userObj.tenantID, false);
-    res.send(objectToSend);
+  var userObj = req.body;
```

Comment:
Please keep the getAllJobsWithResults method and write a new function with this change will do the change
Look for indentation as well